src/Mod/Sketcher/App/SketchObject.h

```diff
@@ -476,6 +478,11 @@ class SketcherExport SketchObject: public Part::Part2DObject
      * \return -1 on error
      */
     int exposeInternalGeometry(int GeoId);
+    template<class GeomType>
+    int exposeInternalGeometryForType(const int GeoId)

```

Comment:
```suggestion
    int exposeInternalGeometryForType([[maybe_unused]] const int GeoId)
```

Reply:
Is this the only complaint from clazy? If so I'll just commit now. Otherwise, I can wait till the morning and resolve things then.

Reply:
I think so, but I got pulled off this to work on something else. No rush.